Realm/RLMArray.mm

```diff
@@ -46,6 +46,14 @@ @implementation RLMArray {
     NSMutableArray *_backingArray;
 }
 
++ (void)load
+{
+    SEL isNSArray = NSSelectorFromString([@[@"is", @"NSArray", @"_", @"_"] componentsJoinedByString:@""]);
+    IMP yes = imp_implementationWithBlock(^(__unused id array) { return YES; });
+    Method isConcurrent = class_getInstanceMethod(NSOperation.class, @selector(isConcurrent));
```

Comment:
It'd be preferable to use one of our own classes / selectors here, since it may cause people to wonder why `NSOperation` is referenced. `-[RLMArray isInvalidated]` looks to have the expected type signature.